### DISCUSSION TITLE
vercel error said that cookies was called outside a request scope.

### DISCUSSION THREAD

**Discussion by Prantor-Hawlader** [amannn/next-intl#1283] [Q&A]
In [example-app-router-without-i18n-routing], locale.ts file cookies was called in server action. When I deployed this code in Vercel. Error occurred:  Unhandled Rejection: Error: cookies was called outside a request scope. How do I handle the cookies in server action?

**Comment by amannn** [comment]
It's definitely working here: https://next-intl-example-app-router-without-i18n-routing.vercel.app/login. If you've adjusted the code for your use case, you might want to review the differences to the original example.

**Comment by Prantor-Hawlader** [comment]
I don't have a src folder in my next.js app. So I moved the i18n.ts file to the root directory, is that the issue? Where should I put that i18n.ts file?

**Reply by Prantor-Hawlader** [reply]
Should I do that according to the next-intl documentation? 
  const withNextIntl = createNextIntlPlugin('./i18n.ts');
